src/extension.ts

```diff
@@ -179,17 +180,33 @@ export function activate(context: vscode.ExtensionContext) {
         return;
       }
 
-      const uri = vscode.Uri.parse(`couchbase:/${node.bucketName}/${node.scopeName}/${node.collectionName}/${documentName}.json`);
-      memFs.writeFile(
-        uri,
-        Buffer.from("{}"),
-        { create: true, overwrite: true }
-      );
+        const uri = vscode.Uri.parse(`couchbase:/${node.bucketName}/${node.scopeName}/${node.collectionName}/${documentName}.json`);
+        try {
```

Comment:
So the intention is to try to retrieve the document with the same key first, then if if returns an error we create a new empty document instead. Please can you add a code comment to describe this behaviour?